net.charlesschaefer.addictiontracker.yml

```
@@ -0,0 +1,95 @@
+id: net.charlesschaefer.addictiontracker
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node22
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: addiction-tracker
+finish-args:
+  - --env=LD_LIBRARY_PATH=/app/lib
+  - --env=FLATPAK_ENV=1
+  - --device=dri
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network # Permission needed to access the network
```

```suggestion
  - --share=network
```

this is self explanatory